### DISCUSSION TITLE
Is it possible to add multiple images in one cell?

### DISCUSSION THREAD

**Discussion by HTMHell** [exceljs/exceljs#2482] [Q&A]
If so, how can I position them horizontally next to each other?

**Comment by Siemienik** [comment]
as I know, you should have possibilities to add how many images as you want (and somewhere you want):

```javascript
worksheet.addImage(imageId2, {
  tl: { col: 1.5, row: 1.5 },
  br: { col: 3.5, row: 5.5 }
});
```
Source: https://github.com/exceljs/exceljs#images

**Comment by HTMHell** [comment]
> as I know, you should have possibilities to add how many images as you want (and somewhere you want):
> 
> ```js
> worksheet.addImage(imageId2, {
>   tl: { col: 1.5, row: 1.5 },
>   br: { col: 3.5, row: 5.5 }
> });
> ```
> 
> Source: https://github.com/exceljs/exceljs#images

That would spread over multiple cells, right? Let's say I have 5 images. How can I put all of them inside one cell (resizing them if they exceed cell's size)

**Comment by Siemienik** [comment]
```JAVASCRIPT
worksheet.addImage(img1, {
  tl: { col: 1.0, row: 1.0 },
  br: { col: 2.0, row: 1.2 }
});

worksheet.addImage(img2, {
  tl: { col: 1.0, row: 1.2 },
  br: { col: 2.0, row: 1.4 }
});


worksheet.addImage(img3, {
  tl: { col: 1.0, row: 1.4 },
  br: { col: 2.0, row: 1.6 }
});

//...
```

should works

**Comment by HTMHell** [comment]
> ```js
> worksheet.addImage(img1, {
>   tl: { col: 1.0, row: 1.0 },
>   br: { col: 2.0, row: 1.2 }
> });
> 
> worksheet.addImage(img2, {
>   tl: { col: 1.0, row: 1.2 },
>   br: { col: 2.0, row: 1.4 }
> });
> 
> 
> worksheet.addImage(img3, {
>   tl: { col: 1.0, row: 1.4 },
>   br: { col: 2.0, row: 1.6 }
> });
> 
> //...
> ```
> 
> should works

So I tired it with 3 images (both rows has 3) and this is what it looks like:
![image](https://user-images.githubusercontent.com/6322901/66751306-68f6fc00-ee97-11e9-914f-8958ef4574bf.png)

This is the data:
![image](https://user-images.githubusercontent.com/6322901/66751330-7a400880-ee97-11e9-93b2-28e62619babd.png)

I was looking to put all images inside the column and maybe even keep proportions, something like this:
![image](https://user-images.githubusercontent.com/6322901/66751544-ea4e8e80-ee97-11e9-9ce5-4794e532d6f6.png)

Appreciate your response

**Comment by Newbie012** [comment]
I'm facing with the same issue...

Can't we just point X images into a specific cell while keeping their ratio?...

**Comment by maltegrosse** [comment]
perhaps merge the images before inserting them. e.g. https://github.com/lukechilds/merge-images
